src/gsi/obsmod.F90

```diff
@@ -631,7 +631,7 @@ module obsmod
   real(r_kind) ,allocatable,dimension(:):: dval
   real(r_kind) ,allocatable,dimension(:):: time_window
 
-  integer(i_kind) ntilt_radarfiles,tcp_posmatch,tcp_box
+  integer(i_kind) ntilt_radarfiles,tcp_posmatch,tcp_box,pmot_dbz,pmot_vr
 
```

Comment:
why is pmot_dbz declared and defined here while it is used in read_dbz_nc?

Reply:
Both pmot_dbz and pmot_vr are defined here when other variables are initialized.  They can be changed at the same time things like the length scales for the thinning, etc. (rmesh_dbz,rmesh_vr,...)are changed.